src/components/Intro.vue

```diff
@@ -19,10 +19,21 @@
         provides the perfect platform for fresh enthusiasts, as well as the
         experienced ones, to code together and compete for ultimate glory.
       </p>
+      <div v-if="!ismobile" :class="$style.buttonContainer">
+        <div :class="$style.link">
+          <router-link to="/login"
+            :class="[$style.linkText]"
+          >
+            <h4>Register / Login</h4>
```

Comment:
I think it's a good idea to show "Dashboard" here and redirect to **/dashboard** if the user is logged in.
To check logged in, use **this.$store.getters.isLoggedIn** getter in a computed property

Reply:
Done